modules/exploits/apple_ios/browser/safari_jit.rb

```diff
@@ -0,0 +1,549 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::Remote::HttpServer::HTML
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Safari JIT Exploit',
```

Comment:
maybe add 'webkit' to the title? that doesn't really add anything, but will help with collisions in the future potentially